pkg/apis/serving/v1alpha2/inferenceservice_types.go

```diff
@@ -60,6 +60,29 @@ type DeploymentSpec struct {
 	// This is the up bound for autoscaler to scale to
 	// +optional
 	MaxReplicas int `json:"maxReplicas,omitempty"`
+	// Activate request/response logging
+	// +optional
+	InferenceLogger *InferenceLogger `json:"inferenceLogger,omitempty"`
+}
+
+type InferenceLoggerType string
+
+const (
+	InferenceLogBoth     InferenceLoggerType = "both"
+	InferenceLogRequest  InferenceLoggerType = "request"
+	InferenceLogresponse InferenceLoggerType = "response"
+)
+
+type InferenceLogger struct {
```

Comment:
Can we write a comment that this feature is experimental? I want to be able to try new additions to our API but signal to customers that the API might change.